README.md

```diff
@@ -259,7 +259,7 @@ Tracer tracer = wfTracerBuilder.build();
 Optionally, you can add custom span-level tags to propagate RED metrics. See [Custom Span-Level Tags for RED Metrics](https://docs.wavefront.com/trace_data_details.html#custom-span-level-tags-for-red-metrics) for details.
 
 ```java
-wfTracerBuilder.redMetricsCustomTagKey("env", "location");
+wfTracerBuilder.redMetricsCustomTagKeys("env", "location");
```

Comment:
sorry, argument also needs to be a set: `wfTracerBuilder.redMetricsCustomTagKeys(new HashSet<String>(Arrays.asList("env", "location")));`

Reply:
Thank you! Updated.